### PR TITLE
BITMAKER-1379: Add comand to return data in csv format

### PR DESCRIPTION
**Pull request by Adriana618**
### Ticket URL:
- https://tasks.bitmaker.dev/issues/1379
### Description:
- Add comand to retrieve data in csv and json format:
> `bitmaker data job <jobID> <spiderID> <projectID> --format <format>` Default format is json,

### CODE REVIEW

File: bm_cli/bm_client.py [hunk 1]
```diff
@@ -179,7 +181,7 @@ def create_spider_job(self, pid, sid, args=[], env_vars=[], tags=[]):
         }
         response = self.post(endpoint, data=data)
         self.check_status(response, 201)
-        return response.json()
+        return response
```
[the remark this answers is not in the document]
Comment (mgonnav) [reply bm_cli/bm_client.py:184]: When creating jobs using the CLI, I'm getting `Error: Cannot create the job for given SID and PID.`. However, the job does create successfully. I guess this is the reason for that.

**Review by mgonnav — CHANGES_REQUESTED** [review]

File: bm_cli/data/job.py [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
 
 SHORT_HELP = "Get data from a job"
+ALLOWED_FORMATS = ["json", "csv"]
 
 
 @click.command(short_help=SHORT_HELP)
 @click.argument("jid", required=True)
 @click.argument("sid", required=True)
 @click.argument("pid", required=False)
+@click.option("-f", "--format", required=False)
```
[the remark this answers is not in the document]
Comment (Adriana618) [reply bm_cli/data/job.py]: Changed.

File: bm_cli/bm_client.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Adriana618) [reply bm_cli/bm_client.py]: Fixed.

File: bm_cli/data/job.py [hunk 3]
```diff
@@ -26,17 +32,11 @@ def bm_command(
...
     bm_client = login()
-    if format is None:
-        format = "json"
-    if format not in ALLOWED_FORMATS:
-        raise ClickException(
-            "Format not supported. Formats that are supported: {}".format(
-                *ALLOWED_FORMATS
-            )
-        )
+    print(format)
```
Comment (mgonnav) [inline bm_cli/data/job.py:39]: Let's remove this print 😅

**Review by mgonnav — APPROVED** [review]
Let's remove the `print` statement and it is good to merge.

File: bm_cli/data/job.py (hunk 3 above)
Comment (Adriana618) [reply bm_cli/data/job.py]: Removed.

**Pull request merged**